zebra-state/src/lib.rs

```diff
@@ -69,6 +69,12 @@ pub struct Config {
 
     /// The maximum number of bytes to use caching data in memory.
     pub memory_cache_bytes: u64,
+
+    /// Whether to use an ephemeral, in-memory store instead of writing to disk.
+    ///
+    /// Set to `false` by default. If this is set to `true`, [`cache_dir`] is ignored and no state
+    /// is written to disk (meaning the entire blockchain may be loaded into memory).
```

Comment:
Sled doesn't actually guarantee that ephemeral databases are created in memory:
https://docs.rs/sled/0.34.3/sled/struct.Config.html#method.temporary

In the current version of sled:

If the configured path is changed, sled uses that path, regardless of the temporary setting:
https://docs.rs/sled/0.34.3/src/sled/config.rs.html#250

If the configured path is the default, and the OS is Linux, sled uses an in-memory temporary directory. Otherwise, it uses an on-disk temporary directory:
https://docs.rs/sled/0.34.3/src/sled/config.rs.html#445

```suggestion
    /// Whether to use an ephemeral database.
    ///
    /// Ephemeral databases are stored in memory on Linux, and in a temporary directory on other OSes.
    ///
    /// Set to `false` by default. If this is set to `true`, [`cache_dir`] is ignored.
```